src/components/FoodList/FoodList.js

```diff
@@ -1,28 +1,51 @@
-import foodData from '../../data/foodData.json'
 import styled from 'styled-components/macro'
+import { useState, useEffect } from 'react'
+import getFood from '../../services/getFood'
 
 export default function FoodList() {
+  const [foodList, setFoodList] = useState([])
+  const [searchInput, setSearchInput] = useState('')
+
+  useEffect(() => {
+    getFood()
+      .then((data) => setFoodList(data))
+      .catch((error) => console.log(error))
+  }, [])
+
+  function handleChange(event) {
+    setSearchInput(event.target.value)
+  }
+
+  const filteredFoodList = foodList.filter((food) =>
+    food.food.toLowerCase().includes(searchInput.toLowerCase())
```

Comment:
I think `food.food` is not very fast to understand. You filter data from a list, maybe you can rename the first `food` to `item`. I am sure you will find a suitable name.